lms/resources/lti_launch.py

```diff
@@ -100,6 +100,22 @@ def resource_link_id(self):
 
         return self._request.POST.get("resource_link_id")
 
+    @property
+    def ext_lti_assignment_id(self):
+        # Canvas SpeedGrader launches don't provide `ext_lti_assignment_id`
+        #
+        # But in order to fix:
+        # * https://github.com/instructure/canvas-lms/issues/1952
+        # * https://github.com/hypothesis/lms/issues/3228
+        #
+        # We include it on the SpeedGrader URL we submit to canvas.
```

Comment:
Is this code really related to those two issues? Those issues are about the incorrect `resource_link_id` in SpeedGrader launches. I think this `ext_lti_assignment_id` property exists for another reason.